cli/printer/text.go

```diff
@@ -171,6 +171,32 @@ func (t *TextPrinter) PrintLoadBalancers(loadBalancers ...*models.LoadBalancer)
 	return nil
 }
 
+func (t *TextPrinter) PrintLoadBalancerHealthCheck(loadBalancer *models.LoadBalancer) error {
+	getEnvironment := func(l *models.LoadBalancer) string {
+		if l.EnvironmentName != "" {
+			return l.EnvironmentName
+		}
+
+		return l.EnvironmentID
+	}
+
+	rows := []string{"LOADBALANCER ID | LOADBALANCER NAME | ENVIRONMENT | TARGET | INTERVAL | TIMEOUT | HEALTHY THRESHOLD | UNHEALTHY THRESHOLD "}
```

Comment:
It is a bit long, but I also feel like we should show something to the user, so they can confirm that the command worked. Maybe it's just enough to not return an error.

However, when you run `loadbalancer healthcheck lb-name` without any flags, it will currently print out the current health check for that load balancer, which seems useful to me.

Is there a different format we could use to display healthcheck information to the user? Should we just drop that line of reasoning altogether?